draftivist_ui/src/components/Campaign.tsx

```diff
@@ -0,0 +1,55 @@
+import * as m from "mithril";
```

Comment:
This file should be in the `components/campaign` dir — maybe that directory change happened after you forked this branch from master? I'm a little confused since I pulled down this branch and `index.ts` is still linking to the `components/campaign/Campaign.tsx` component. Does your index.ts look different?

Reply:
i have it in both places actually. Let me try to merge in main again and see if that resolves it.